sormas-api/src/main/resources/strings.properties

```diff
@@ -1125,42 +1141,77 @@ messageEventParticipantDearchived = Event participant has been de-archived
 messageCountCasesNotRestored = %s cases not restored. UUIDs of cases not restored: %s
-messageCountContactsNotDeleted = %s contacts not deleted. UUIDs of contacts not deleted: %s
+messageCountContactsNotDeleted = %s contacts not deleted: %s
+messageCountContactsNotDeletedAccessDeniedReason = %s contacts not deleted because they are not in jurisdiction or owned: %s
+messageCountContactsNotDeletedSormasToSormasReason = %s contacts not deleted because they were already shared with Sormas to Sormas: %s
+messageCountContactsNotLinkableAccessDeniedReason = %s contacts not linked because the contact is not editable anymore or the user is missing the rights: %s
 messageCountContactsNotRestored = %s contacts not restored. UUIDs of contact not restored: %s
+messageCountFollowUpsNotCancelled = %s contacts' follow-up not cancelled: %s
+messageCountFollowUpsNotCancelledAccessDeniedReason = %s contacts' follow-up not cancelled because the contact is not editable anymore or is outside the user's jurisdiction: %s
+messageCountFollowUpsNotSetToLost = %s contacts' follow-up not set to lost: %s
+messageCountFollowUpsNotSetToLostAccessDeniedReason = %s contacts' follow-up not set to lost because the contact is not editable anymore or is outside the user's jurisdiction: %s
+messageCountEntitiesNotArchived = %s entities can not be archived: %s
+messageCountEntitiesNotArchivedAccessDeniedReason = %s infrastructure entities can not be archived because the entities are used in other infrastructure data: %s
+messageCountEntitiesNotDearchived = %s entities can not be dearchived: %s
+messageCountEntitiesNotDearchivedAccessDeniedReason = %s infrastructure entities can not be dearchived because the entities have archived parent infrastructure: %s
+messageCountEntitiesNotEdited = %s entities can not be edited because they are in a different jurisdiction or were already archived.: %s
+messageCountEntitiesNotEditedAccessDeniedReason = %s entities not edited because the entity is not editable anymore: %s
+messageCountEntitiesNotSent = %s entities can not be sent to the reporting tool: %s
+messageCountEntitiesNotSentAccessDeniedReason = %s entities can not be sent to the reporting tool because the user does not have the proper rights: %s
+messageCountEntitiesNotSentExternalReason = %s entities can not be sent to the reporting tool because the communication with the reporting tool failed: %s
+messageCountEventsNotArchivedExternalReason = %s events not archived because the communication with the reporting tool failed: %s
+messageCountEventsNotDearchivedExternalReason = %s events not dearchived because the communication with the reporting tool failed: %s
+messageCountEventsNotLinked = %s events can not be linked or they are already linked to the event group: %s
+messageCountEventsNotLinkedAccessDeniedReason = %s events can not be linked to the event group because is not allowed to link events from another region to an event group.: %s
```

Comment:
```suggestion
messageCountEventsNotLinkedAccessDeniedReason = %s events can not be linked to the event group because it's not allowed to link events from another region to an event group: %s
```